CoolBook/Controllers/AuthorsController.cs

```diff
@@ -20,9 +21,54 @@ public AuthorsController(CoolBookContext context)
             _context = context;
         }
 
+        public class AuthorExtand
+        {
+            public int Id { get; set; }
+            public string Name { get; set; }
+
+            [Required, Display(Name = "Birth Date"), DataType(DataType.Date)]
+            public DateTime BirthDate { get; set; }
+            public Gender Gender { get; set; }
+            public string Country { get; set; }
+
+            [Display(Name = "Books Count")]
+            public int BookCount { get; set; }
+        }
+
         // GET: Authors
         public async Task<IActionResult> Index()
         {
+            return View(await _context.Author.Join(
+                _context.Book,
+                author => author.Id,
+                book => book.AuthorId,
+                (author, book) => new
+                {
+                    Id = author.Id,
+                    Name = author.Name,
+                    BirthDate = author.BirthDate,
+                    Gender = author.Gender,
+                    Country = author.Country,
+                    BookId = book.Id
+                })
+                .GroupBy(x => new
```

Comment:
move the group by down and then you don't need the new?

Reply:
this is how the group by works